src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { Status } from '../../types/Status';
+import { ErrorMessages } from '../../types/ErrorMessages';
+import { deleteTodos } from '../../api/todos';
+
+interface Props {
+  todos: Todo[];
+  selectedStatus: Status;
+  setSelectedStatus: (status: Status) => void;
+  setTodos: React.Dispatch<React.SetStateAction<Todo[]>>;
+  setErrorMessage: (message: string) => void;
+  setLoadingIds: React.Dispatch<React.SetStateAction<number[]>>;
+}
+
+export const Footer: React.FC<Props> = ({
+  todos,
+  selectedStatus,
+  setSelectedStatus,
+  setTodos,
+  setErrorMessage,
+  setLoadingIds,
+}) => {
+  const todosLeft = todos.filter(todo => !todo.completed).length;
+  const hasCompleted = todos.filter(todo => todo.completed);
```

Comment:
```suggestion
  const completedTodos = todos.filter(todo => todo.completed);
```
It's not a boolean, so to avoid confusion rename it